util/sdkutil/string.go

```diff
@@ -30,3 +31,13 @@ func FormatDec(d sdk.Dec) string {
 func FormatDecCoin(c sdk.DecCoin) string {
 	return fmt.Sprintf("%s %s", FormatDec(c.Amount), c.Denom)
 }
+
+func GenerateString(length uint) string {
```

Comment:
Should add code coverage for this new function

Reply:
I think a simple unit test will be enough - this function is a helper function for tests.